wherescape/connectors/hubspot/collect_data.py

```diff
@@ -0,0 +1,55 @@
+import logging
+from datetime import datetime
+from ...wherescape import WhereScape
+from .process_data import hubspot_process_results
+
+""" 
+this module retrieves the data from Wherescape
+"""
+
+
+def hubspot_load_data():
+    """
+    This method collects all the data from a table and sends it to be processed.
+    """
+    start_time = datetime.now()
+    logging.info("connecting to WhereScape")
+    wherescape_instance = WhereScape()
+    logging.info(
+        "Start time: %s for hubspot_load_data"
+        % start_time.strftime("%Y-%m-%d %H:%M:%S")
+    )
+    logging.info("post load")
+    table_name = f"{wherescape_instance.schema}.{wherescape_instance.table}"
+    sql = f"select * from {table_name}"
+
+    result = wherescape_instance.query_target(sql)
+    access_token = hubspot_get_token(wherescape_instance, table_name)
+    column_names = wherescape_instance.get_columns()[0]
+
+    if len(result) > 0:
+        hubspot_process_results(access_token, result, column_names, table_name)
+        logging.info("hubspot update done")
+
+
+def hubspot_get_token(wherescape_instance: WhereScape, table_name: str):
+    """
+    This method allows for different access tokens based on the environment mentioned
+    in the table name. If no environment is found this way. It will use the basic
+    parameter name to retrieve the token
+    """
+    parameter_name = "hubspot_acces_token"
+    table_words = table_name.split("_")
+
+    logging.info("retrieving access_token")
+
+    for word in table_words:
+        environment_parameter = parameter_name + "_" + word
+        access_token = wherescape_instance.read_parameter(environment_parameter)
+
+        if access_token:
```

Comment:
I can't be certain where the name of the environment is in the table name since others might order the words differently. So I just check every word without warning. 
giving info on what parametername the token is taken from should still help if something is wrong on that end, but I didn't think it requires a warn